docker-compose.yml

```diff
@@ -1,21 +1,21 @@
 version: '2'
 
 services:
-  geotrellis:
-    image: gtserver
+  service:
+    image: quay.io/geotrellis/gtsite-service
     build:
-      context: ./geotrellis
+      context: ./service
       dockerfile: Dockerfile
     ports:
       - "9090:9090"
     volumes:
       - "${HOME}/.ivy2:/root/.ivy2"
       - "${HOME}/.m2:/root/.m2"
       - "${HOME}/.sbt:/root/.sbt"
-  jekyll:
-    image: jekyll-nginx
+  static:
```

Comment:
I think that it would be better to separate the actual creation of the static site into a separate container and have the Nginx container only responsible for serving the results of the static site build. This is how things currently happen in RF, except that the static build pipeline is using Webpack.